client/lib/terms/constants.js

```diff
@@ -13,3 +13,9 @@ module.exports.action = keyMirror( {
 	FETCH_TAGS: null,
 	SET_CATEGORY_SELECTED_ITEMS: null
 } );
+
+module.exports.defaultNonHierarchicalQuery = {
+	number: module.exports.MAX_TAGS,
+	order_by: 'count',
+	order: 'DESC'
+};
```

Comment:
I considered bringing this file up to date with ES6 but didn't want to also change the existing imports and usage in this PR.